docs/api-docs/storefront/widgets/widgets-overview.md

```diff
@@ -339,9 +339,9 @@ Widgets are rendered on the storefront as a data tag in the HTML.
 
 | Name | Definition |
 |--|--|
-| Widgets | Widgets are the units of content to be placed on specific pages in a Stencil theme. Each widget is composed of a widget configuration and a widget template. *There is a limit of `10,000` widgets per store.* |
+| Widgets | Widgets are the units of content to be placed on specific pages in a Stencil theme. Each widget is composed of a widget configuration and a widget template. *There is a limit of `100,000` widgets per store.* |
```

Comment:
@christensenep 

- please verify the new quantities on each edited line
- please clarify whether the limit is per _store_ or per _storefront_

many thanks.